share/commands/agent-inventory

```diff
@@ -24,5 +24,16 @@
 # @man *-R*: run the agent in completely unparsed mode, with no return code of 1 in case of error. A little faster.
 # @man +
 # @man *-m*: run the agent with multiline output
+# @man +
+# @man *-f*: run the agent even if it is disabled
+
+OPTS=""
+argcount=$#
+# Process all options in order (getopts doesn't set OPTIND in case of error)
+for OPTIND in `seq 1 ${argcount}`
+do
+  getopts ":f" opt # ignore getopt errors
+  [ "${opt}" = "f" ] && OPTS="-D ignore_disable_inventory"
```

Comment:
There is already a "-D" option in the command call below. As Alexis pointed out, cf-agent doesn't support multiple -D options, we need to use the -D class1,class2 syntax.